### PR TITLE
feat: adding put, post, and delete example

### PR DESCRIPTION
**Pull request by aaron-peloquin**
Adding another example for how to reply with function components, since it's less spelled out than with object-oriented.

I wasn't sure how we listen to POSTed variables, so I didn't include that, but I want to add it before merging

### CODE REVIEW

File: examples/routes/get-put-post-delete.js [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+          httpRequest.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded')
+          httpRequest.send()
+
+          /* DELETE */
+          httpRequest = new XMLHttpRequest()
+          httpRequest.onload = showResponse
+          httpRequest.open('DELETE', '/get-put-post-delete')
+          httpRequest.setRequestHeader('Content-Type', 'application/x-www-form-urlencoded')
+          httpRequest.send()
+        </script>
```
Comment (aaron-peloquin) [inline examples/routes/get-put-post-delete.js]: over 90% of this file is just setting up and making ajax calls :(

Comment (aaron-peloquin) [reply examples/routes/get-put-post-delete.js]: let me know if you can think of a better way to show this off

**Comment by aaron-peloquin** [comment]
I really want to take the `number` I'm sending to the example requests and run some math on it, then return it in the response body but I don't know how to grab the `Form Data` that I sent 😞

**Comment by justinjordan** [comment]
That I'm still working on. It's still kind of broken since I yanked out Hapi.

**Review by justinjordan — COMMENTED** [review]
I don't think we need to show the frontend scripts in the get request. Maybe just keep the get request as simple as the other methods.

**Comment by aaron-peloquin** [comment]
Ooh, that's a good point!

**Comment by aaron-peloquin** [comment]
@sXule is this what you were thinking?

**Comment by justinjordan** [comment]
No sorry, I meant omit the `httpRequest` stuff just because it's kind of confusing. Also because people could be using axios or fetch, and front-end is outside the scope of the library.

**Comment by aaron-peloquin** [comment]
👍

**Pull request merged**